src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -177,6 +197,21 @@ private void setWindowMinSize() {
     }
 
     /**
+     * Sets the default theme based on user preferences.
+     */
+    private void setDefaultTheme(UserPrefs prefs, Scene scene) {
+        scene.getStylesheets().addAll(prefs.getThemeSettings().getTheme(),
+                prefs.getThemeSettings().getThemeExtensions());
+    }
+
+    /**
+     * Returns the current theme applied.
+     */
+    ThemeSettings getCurrentThemeSetting() {
+        return new ThemeSettings(scene.getStylesheets().get(0), scene.getStylesheets().get(1));
```

Comment:
Can we define constants for these numbers instead of `0` or `1`? It makes it easier to understand what is going on in this line.

Reply:
Updated! 👍✔